mesonbuild/msetup.py

```diff
@@ -293,6 +293,8 @@ def _generate(self, env: environment.Environment, capture: bool, vslite_ctx: dic
                 with open(fname, 'w', encoding='utf-8') as f:
                     json.dump(data, f)
 
+                mlog.log("meson setup completed")  # Display timestamp
```

Comment:
Shouldn't this be log_timestamp()?

Reply:
It could. But since it's already inside a `if self.options.profile` block, it doesn't really matter..

Reply:
ah, didn't notice it's in that block when reading the diff. Makes sense then.